js/plugins/code/base-text-element.js

```
@@ -959,7 +965,201 @@ class BaseTextElement extends HTMLElement {
+                    event.preventDefault();
+                    this.selectedPasteLinkIndex = Math.min(2, this.selectedPasteLinkIndex + 1);
+                    this.updatePasteLinkMenuSelection();
+                    return;
+                case 'ArrowUp':
+                    event.preventDefault();
+                    this.selectedPasteLinkIndex = Math.max(0, this.selectedPasteLinkIndex - 1);
+                    this.updatePasteLinkMenuSelection();
+                    return;
+                case 'Enter':
+                    event.preventDefault();
+                    const types = ['url', 'bookmark', 'embed'];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)